nncf/common/composite_compression.py

```diff
@@ -280,6 +281,62 @@ def prepare_for_export(self) -> None:
             stripped_model = ctrl.strip_model(stripped_model)
         self._model = stripped_model
 
+    def prepare_for_inference(self, make_model_copy: bool = True) -> TModel:
+        """
+        Prepare compressed model for inference by converting NNCF modules to native format.
+
+        :param make_model_copy: `True` means that a copy of the model will be modified.
+            `False` means that the original model in the controller will be changed and
+            no further compression actions will be available. Defaults to True.
+
+        :return: Modified model.
+        """
+        # Because of the differences in model preparation for Torch and TF backends, it is not possible to use
+        # the same pipeline. TF transformations (expect quantization) creates copy of the model,
+        # torch transformations can modify existed instance of the model.
+
+        model = self.model
+
+        backend = get_backend(model)
+        if backend is BackendType.TENSORFLOW:
+            from nncf.tensorflow.quantization.algorithm import QuantizationController
```

Comment:
There is need specific pipeline for prepare_for_inference for TF backend, because TF transformations creates copy of the model, expect of quantization that modify existed instance.